Source/Mod/Data/SaveManager.cs

```diff
@@ -69,6 +84,38 @@ internal void NewSave(string? name = null)
 		}
 	}
 
+	[DisallowHooks]
```

Comment:
`[DisallowHooks]` is only required to mark things inside the `Celeste64` namespace. Every other namespace isn't included anyway.